### ISSUE TITLE
Time created and updated on some notes are 1 hour ahead

### ISSUE THREAD

**Issue by Goldmaster** [laurent22/joplin#10928]
### Operating system

Windows

### Joplin version

1.4.1

### Desktop version info

Joplin 3.0.15 (prod, win32)

Client ID: 19cd933e2d32416e851c331a1291152b
Sync Version: 3
Profile Version: 47
Keychain Supported: Yes

Revision: 598677b

Backup: 1.4.1

### Current behaviour

For some reason, some notes are showing in the desktop as being an hour ahead ![image](https://github.com/user-attachments/assets/69c82459-8583-4c2a-9cb6-f70a580213d7)

However, in the md file, you can see the time is an hour behind. 
![image](https://github.com/user-attachments/assets/2910ba7c-00ff-4186-8079-438881ca69c6)

### Expected behaviour

If i updated the md files with the correct time, I expect Joplin desktop to sync the new changes using the delete local data and download. which display a correct time.

### Logs

https://privatebin.net/?681449bff9faecf2#DfrjWXiXqEaiPdvUfx6sgahwF2HEJrWgxRU5LPD4yWSK

**Comment by tomasz1986** [comment]
The time as seen in the MD files uses GMT. That's what the "Z" stands for at the end. On the other hand, Joplin probably uses your local time in the GUI.

**Comment by Goldmaster** [comment]
> Joplin probably uses your local time in the GUI.

That's strange as Windows 11 is set to EN-GB so they should show as the same time.

**Comment by tomasz1986** [comment]
It is summer time in the UK currently, so it's GMT+1.

**Comment by Goldmaster** [comment]
So it is working correctly?

We just need to get rid of this stupid back and forth with changing clocks.

**Comment by dpoulton-github** [comment]
> So it is working correctly?

TL/DR: **Yes**

The "z" at the end of the date in the MD file indicates "Zulu Time" which is another way of saying UTC. This is often also referred to as GMT although UTC succeeded GMT. Computers store dates/times as UTC and a computer's locale setting converts this to local time. If you change your locale the UTC date in the MD will be the same but the displayed local date will change.

Below is the **same note** (19:22 UTC) when the locale is set as (1) Europe/London and (2) America/Anchorage

![europe_london](https://github.com/user-attachments/assets/59e91c27-4b1f-4031-bb83-54762c86a448)
*1 - Europe/London - BST - British Summer Time (UTC+1)*

![america_anchorage](https://github.com/user-attachments/assets/055992dc-93cd-44e1-b25b-7bf5e31b29a6)
*2 - America/Anchorage - AKDT - Alaska Daylight Time (UTC-8)*
